src/lib/PointerTracker/index.ts

```diff
@@ -1,3 +1,4 @@
+import { bind } from '../util';
```

Comment:
Just for me: We are doing extension-less imports because webpack, right?

Reply:
A vscode extension I used added this import for me 😄.

Not wedded to extension-less imports. Any feels here @developit @kosamari?

Reply:
FWIW I tend to use `.js` extension. TypeScript automatically prefers `.ts` files with the same name if they exist. But no strong feels on my side.

Reply:
TBH it doesn't matter to me. It's extremely uncommon to use extensions in Webpack, though.

Reply:
Drive-by opinion: explicit > implicit

Reply:
>  It's extremely uncommon to use extensions in Webpack

Probably the strongest argument so far, considering one of our goals was to meet developers where they are currently at.